feat(bridge-ui-v2): NFT Bridge UI initial setup

KorbinianK opened the pull request:
- Fetching NFT images, 
- detecting contract types,
- added navigation buttons based on design
- updated dependencies

dantaik commented:
@KorbinianK there is a build failure - https://github.com/taikoxyz/taiko-mono/actions/runs/5644160065/job/15287386711

KorbinianK commented:
> @KorbinianK there is a build failure - https://github.com/taikoxyz/taiko-mono/actions/runs/5644160065/job/15287386711

Fixed the issues. Added the svlete:check to lefthook as well now, we should run into these types of errors before we try a build.
